scripts/ss.sh

```diff
@@ -1,17 +1,25 @@
 #!/bin/bash
 set -e
-trap "cleanup $? $LINENO" EXIT
+DEBUG="yes"
+if [ "${DEBUG}" == "NO" ]; then
+  trap "cleanup $? $LINENO" EXIT
+fi
 
 ## Deployment Variables
```

Comment:
Is there a reason we are only giving users the option to deploy with rDNS instead of a FQDN? Just curious.